google-cloud-logging/src/main/java/com/google/cloud/logging/Instrumentation.java

```diff
@@ -37,7 +37,9 @@ public final class Instrumentation {
   public static final String INSTRUMENTATION_NAME_KEY = "name";
   public static final String INSTRUMENTATION_VERSION_KEY = "version";
   public static final String JAVA_LIBRARY_NAME_PREFIX = "java";
-  public static final String DEFAULT_INSTRUMENTATION_VERSION = "UNKNOWN";
+  // {x-version-update-start:google-cloud-logging:current}
+  public static final String DEFAULT_INSTRUMENTATION_VERSION = "1.0.0";
```

Comment:
Are you sure 1.0.0 is a good default? Would 0.0.0 work?

Reply:
it supposed to change after release